src/canvas/Point.tsx

```diff
@@ -19,6 +19,10 @@ export class Point implements IPoint {
         return this._x === o.getX() && this._y === o.getY();
     }
 
+    isWithinRadius = (o: Point, radius: number): boolean => {
+        return Math.pow((o.getX() - this._x), 2) + Math.pow((o.getY() - this._y), 2) < Math.pow(radius, 2);
```

Comment:
Also, in code I tend to prefer radius * radius to Math.pow(radius, 2) to avoid an unnecessary function call.

Reply:
I don't think the function call is that big of a deal in java script, but I will change it.